internal/types/types.go

```diff
@@ -189,3 +192,31 @@ func deepCopy(value any) any {
 		panic(fmt.Sprintf("types.deepCopy: unsupported type: %[1]T (%[1]v)", value))
 	}
 }
+
+// JSONSyntax formats a FerretDB value as it's JSON counterpart.
```

Comment:
can you please write a note about how it differs from `fjson` package, which also makes the thing `from-to`? and clarify why `fjson` cannot be used.